prow/jobs/tools/cmd/command/query_golang_ecr_helper.go

```diff
@@ -0,0 +1,77 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/aquasecurity/go-version/pkg/semver"
+	"github.com/aws-controllers-k8s/test-infra/command/queryecrpublic"
+	"gopkg.in/yaml.v3"
+)
```

Comment:
Need to be reordered using standards mentioned in https://go.dev/wiki/CodeReviewComments#imports